docs/GL5/SRI/Indexation/cheat-sheet.md

```diff
@@ -8,15 +8,15 @@ authors:
 
 Author [@medbensalah](https://github.com/medbensalah)
 
-## Indexation
+# Indexation
 
 ### Definition
 
 Processus permettant de construire un ensemble d'éléments
-**clés** permettant de caractériser le contenu d’un
+_clés_ permettant de caractériser le contenu d’un
```

Comment:
**** is the official markdown for bold  
https://www.markdownguide.org/basic-syntax/#bold